### PR TITLE
wtf: 0.31.0 -> 0.32.0

### PR DESCRIPTION
**Pull request by avdv**
Semi-automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. 

meta.description for wtf is: "The personal information dashboard for your terminal"

meta.homepage for wtf is: "https://wtfutil.com/"

meta.changelog for wtf is: ""

###### Updates performed
- Golang update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/wtfutil/wtf/releases/tag/v0.32.0)

- [Compare changes on GitHub](https://github.com/wtfutil/wtf/compare/v0.31.0...v0.32.0)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 0.32.0 with grep in /nix/store/33ky4cyrz4qwzwcax2zmz9zz118gl3ll-wtf-0.32.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

0 x86_64-linux rebuild(s)
0 i686-linux rebuild(s)
0 x86_64-darwin rebuild(s)
0 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
wtf
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Build yourself:
```
nix-build -A wtf https://github.com/avdv/nixpkgs/archive/946d8304bc9b0a3153fd60247a54a6f56aa0fc0b.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/33ky4cyrz4qwzwcax2zmz9zz118gl3ll-wtf-0.32.0
ls -la /nix/store/33ky4cyrz4qwzwcax2zmz9zz118gl3ll-wtf-0.32.0/bin
```

---

</details>
<br/>

### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>wtf</li>
  </ul>
</details>

---

###### Maintainer pings

cc @kalbasit @kalbasit for testing.

### CODE REVIEW

**Review by stigtsp — CHANGES_REQUESTED** [review]
Build OK on `x64 linux`, but build FAILS on `x64 darwin`:

```
go: downloading github.com/huandu/xstrings v1.2.0
go: downloading github.com/fatih/color v1.7.0
go: downloading github.com/mndrix/tap-go v0.0.0-20170113192335-56cca451570b
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d
installing
hash mismatch in fixed-output derivation '/nix/store/cbn96y1r2h21a9zrsrr1ffgm25nlbcgg-wtf-0.32.0-go-modules':
  wanted: sha256:0l1q29mdb13ir7n1x65jfnrmy1lamlsa6hm2jagf6yjbm6wf1kw4
  got:    sha256:0p4cy7snxkx5in7c0nsy6bm0f1gn297rzwalm5rdfvhgdlb102nw
cannot build derivation '/nix/store/hfkjz60aksbgy7x9xmrbbbi30rjmzmp7-wtf-0.32.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/hfkjz60aksbgy7x9xmrbbbi30rjmzmp7-wtf-0.32.0.drv' failed
```

**Comment by avdv** [comment]
> Build OK on `x64 linux`, but build FAILS on `x64 darwin`:

Do you know why this happens? Is package resolution platform dependent in Go?

**Comment by avdv** [comment]
> Build OK on `x64 linux`, but build FAILS on `x64 darwin`:
> 
> ```
> go: downloading github.com/huandu/xstrings v1.2.0
> go: downloading github.com/fatih/color v1.7.0
> go: downloading github.com/mndrix/tap-go v0.0.0-20170113192335-56cca451570b
> go: downloading github.com/hashicorp/hcl v1.0.0
> go: downloading github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d
> installing
> hash mismatch in fixed-output derivation '/nix/store/cbn96y1r2h21a9zrsrr1ffgm25nlbcgg-wtf-0.32.0-go-modules':
>   wanted: sha256:0l1q29mdb13ir7n1x65jfnrmy1lamlsa6hm2jagf6yjbm6wf1kw4
>   got:    sha256:0p4cy7snxkx5in7c0nsy6bm0f1gn297rzwalm5rdfvhgdlb102nw
> cannot build derivation '/nix/store/hfkjz60aksbgy7x9xmrbbbi30rjmzmp7-wtf-0.32.0.drv': 1 dependencies couldn't be built
> error: build of '/nix/store/hfkjz60aksbgy7x9xmrbbbi30rjmzmp7-wtf-0.32.0.drv' failed
> ```

I have no idea whether this is the right thing to do (it feels wrong...), but I changed the vendor SHA based on the platform.

**Comment by stigtsp** [comment]
> I have no idea whether this is the right thing to do (it feels wrong...), but I changed the vendor SHA based on the platform.

I'm not that experienced with Go, so unsure myself. But I share your feeling that it seems wrong.

**Comment by avdv** [comment]
Here's the list of go modules which are downloaded when building the go-modules derivation on x64 (from https://logs.nix.ci/?attempt_id=e585fe01-1c04-49e1-aa92-1f99613dedb6&key=nixos%2Fnixpkgs.98622):
```
go: downloading github.com/olebedev/config v0.0.0-20190528211619-364964f3a8e4
go: downloading github.com/briandowns/openweathermap v0.0.0-20180804155945-5f41b7c9d92d
go: downloading github.com/google/go-github/v26 v26.1.3
go: downloading github.com/mmcdole/gofeed v1.0.0
go: downloading github.com/rivo/tview v0.0.0-20200108161608-1316ea7a4b35
go: downloading github.com/radovskyb/watcher v1.0.7
go: downloading github.com/ovh/cds v0.0.0-20200131143542-5e69464c6598
go: downloading github.com/docker/docker v1.13.1
go: downloading github.com/digitalocean/godo v1.42.1
go: downloading github.com/microsoft/azure-devops-go-api/azuredevops v0.0.0-20191014190507-26902c1d4325
go: downloading github.com/gdamore/tcell v1.3.0
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/olekukonko/tablewriter v0.0.4
go: downloading github.com/google/go-querystring v1.0.0
go: downloading google.golang.org/api v0.30.0
go: downloading github.com/mattn/go-runewidth v0.0.7
go: downloading github.com/zmb3/spotify v0.0.0-20191010212056-e12fb981aacb
go: downloading github.com/PagerDuty/go-pagerduty v0.0.0-20191002190746-f60f4fc45222
go: downloading github.com/adlio/trello v1.7.0
go: downloading github.com/shirou/gopsutil v2.20.7+incompatible
go: downloading github.com/google/uuid v1.1.1
go: downloading github.com/mmcdole/goxpp v0.0.0-20181012175147-0068e33feabf
go: downloading github.com/alecthomas/chroma v0.8.0
go: downloading github.com/gdamore/encoding v1.0.0
go: downloading golang.org/x/sys v0.0.0-20200803210538-64077c9b5642
go: downloading github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e
go: downloading golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
go: downloading github.com/rivo/uniseg v0.1.0
go: downloading github.com/andygrunwald/go-gerrit v0.0.0-20190825170856-5959a9bf9ff8
go: downloading golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
go: downloading github.com/lucasb-eyer/go-colorful v1.0.2
go: downloading github.com/pkg/profile v1.5.0
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading google.golang.org/appengine v1.6.6
go: downloading github.com/hekmon/transmissionrpc v0.0.0-20190525133028-1d589625bacd
go: downloading github.com/PuerkitoBio/goquery v1.5.0
go: downloading github.com/docker/go-connections v0.4.0
go: downloading cloud.google.com/go v0.62.0
go: downloading github.com/jessevdk/go-flags v1.4.0
go: downloading gopkg.in/yaml.v2 v2.3.0
go: downloading github.com/wtfutil/spotigopher v0.0.0-20191127141047-7d8168fe103a
go: downloading k8s.io/client-go v10.0.0+incompatible
go: downloading github.com/logrusorgru/aurora v0.0.0-20190803045625-94edacc10f9b
go: downloading github.com/nicklaw5/helix v0.5.9
go: downloading golang.org/x/net v0.0.0-20200707034311-ab3426394381
go: downloading code.cloudfoundry.org/bytefmt v0.0.0-20190819182555-854d396b647c
go: downloading github.com/golang/protobuf v1.4.2
go: downloading k8s.io/apimachinery v0.0.0-20190223094358-dcb391cde5ca
go: downloading google.golang.org/protobuf v1.25.0
go: downloading github.com/docker/distribution v2.7.1+incompatible
go: downloading github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d
go: downloading golang.org/x/text v0.3.3
go: downloading github.com/dlclark/regexp2 v1.2.0
go: downloading github.com/opencontainers/go-digest v1.0.0-rc1
go: downloading github.com/andybalholm/cascadia v1.0.0
go: downloading k8s.io/api v0.0.0-20181204000039-89a74a8d264d
go: downloading github.com/danwakefield/fnmatch v0.0.0-20160403171240-cbb64ac3d964
go: downloading github.com/docker/go-units v0.4.0
go: downloading github.com/godbus/dbus v4.1.0+incompatible
go: downloading github.com/google/gofuzz v1.0.0
go: downloading github.com/gophercloud/gophercloud v0.5.0
go: downloading github.com/zorkian/go-datadog-api v2.29.0+incompatible
go: downloading github.com/gogo/protobuf v1.2.1
go: downloading k8s.io/klog v0.2.0
go: downloading github.com/hekmon/cunits v2.0.1+incompatible
go: downloading github.com/VictorAvelar/devto-api-go v1.0.0
go: downloading github.com/wtfutil/todoist v0.0.2-0.20191216004217-0ec29ceda61a
go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
go: downloading sigs.k8s.io/yaml v1.1.0
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/stretchr/testify v1.6.1
go: downloading github.com/cenkalti/backoff v2.2.1+incompatible
go: downloading github.com/googleapis/gnostic v0.1.0
go: downloading github.com/go-ole/go-ole v1.2.4
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading github.com/imdario/mergo v0.3.8
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/gregjones/httpcache v0.0.0-20190212212710-3befbb6ad0cc
go: downloading github.com/xanzy/go-gitlab v0.33.0
go: downloading gotest.tools v2.2.0+incompatible
go: downloading gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
go: downloading github.com/peterbourgon/diskv v2.0.1+incompatible
go: downloading github.com/docker/docker-credential-helpers v0.6.3
go: downloading google.golang.org/grpc v1.29.1
go: downloading github.com/google/go-cmp v0.5.1
go: downloading github.com/Azure/go-autorest v11.1.2+incompatible
go: downloading gopkg.in/inf.v0 v0.9.1
go: downloading github.com/google/btree v1.0.0
go: downloading github.com/json-iterator/go v1.1.7
go: downloading go.opencensus.io v0.22.4
go: downloading github.com/hashicorp/go-retryablehttp v0.6.4
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/googleapis/gax-go/v2 v2.0.5
go: downloading github.com/Microsoft/go-winio v0.4.14
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading golang.org/x/time v0.0.0-20191024005414-555d28b269f0
go: downloading github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
go: downloading google.golang.org/genproto v0.0.0-20200804131852-c06518451d9c
go: downloading github.com/ovh/venom v0.25.0
go: downloading github.com/sguiheux/go-coverage v0.0.0-20190710153556-287b082a7197
go: downloading github.com/sirupsen/logrus v1.4.2
go: downloading github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: downloading github.com/fatih/color v1.7.0
go: downloading github.com/fsamin/go-dump v1.0.9
go: downloading gopkg.in/AlecAivazis/survey.v1 v1.7.1
go: downloading github.com/spf13/cobra v0.0.5
go: downloading github.com/go-gorp/gorp v2.0.0+incompatible
go: downloading github.com/pborman/uuid v1.2.0
go: downloading github.com/mattn/go-colorable v0.1.6
go: downloading github.com/mitchellh/hashstructure v0.0.0-20170609045927-2bca23e0e452
go: downloading github.com/maruel/panicparse v1.3.0
go: downloading github.com/ovh/cds/sdk/interpolate v0.0.0-20190319104452-71125b036b25
go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.2
go: downloading github.com/mattn/go-isatty v0.0.12
go: downloading github.com/eapache/go-resiliency v1.1.0
go: downloading github.com/spf13/afero v1.2.2
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: downloading github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
go: downloading github.com/huandu/xstrings v1.2.0
go: downloading github.com/aokoli/goutils v1.1.0
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d
go: downloading github.com/mndrix/tap-go v0.0.0-20170113192335-56cca451570b
```
@stigok  Do you have the list for Darwin at hand? Are there any differences?

**Comment by stigtsp** [comment]
@aanderse can you trigger a darwin build for this PR?

**Comment by stigtsp** [comment]
The order of the output is different, but content is the same when sorted.

Build output from Darwin (the previous commit):
```
go: downloading github.com/alecthomas/chroma v0.8.0
go: downloading github.com/docker/docker v1.13.1
go: downloading github.com/gdamore/tcell v1.3.0
go: downloading github.com/xanzy/go-gitlab v0.33.0
go: downloading k8s.io/client-go v10.0.0+incompatible
go: downloading github.com/google/go-github/v26 v26.1.3
go: downloading github.com/docker/docker-credential-helpers v0.6.3
go: downloading github.com/rivo/tview v0.0.0-20200108161608-1316ea7a4b35
go: downloading github.com/pkg/profile v1.5.0
go: downloading github.com/radovskyb/watcher v1.0.7
go: downloading github.com/ovh/cds v0.0.0-20200131143542-5e69464c6598
go: downloading github.com/olekukonko/tablewriter v0.0.4
go: downloading github.com/mattn/go-runewidth v0.0.7
go: downloading k8s.io/apimachinery v0.0.0-20190223094358-dcb391cde5ca
go: downloading golang.org/x/text v0.3.3
go: downloading golang.org/x/sys v0.0.0-20200803210538-64077c9b5642
go: downloading github.com/olebedev/config v0.0.0-20190528211619-364964f3a8e4
go: downloading github.com/hashicorp/go-retryablehttp v0.6.4
go: downloading github.com/VictorAvelar/devto-api-go v1.0.0
go: downloading github.com/gdamore/encoding v1.0.0
go: downloading k8s.io/klog v0.2.0
go: downloading github.com/gogo/protobuf v1.2.1
go: downloading github.com/imdario/mergo v0.3.8
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/andygrunwald/go-gerrit v0.0.0-20190825170856-5959a9bf9ff8
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
go: downloading github.com/logrusorgru/aurora v0.0.0-20190803045625-94edacc10f9b
go: downloading github.com/briandowns/openweathermap v0.0.0-20180804155945-5f41b7c9d92d
go: downloading github.com/zorkian/go-datadog-api v2.29.0+incompatible
go: downloading github.com/docker/go-connections v0.4.0
go: downloading github.com/rivo/uniseg v0.1.0
go: downloading golang.org/x/net v0.0.0-20200707034311-ab3426394381
go: downloading k8s.io/api v0.0.0-20181204000039-89a74a8d264d
go: downloading github.com/google/gofuzz v1.0.0
go: downloading golang.org/x/time v0.0.0-20191024005414-555d28b269f0
go: downloading golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
go: downloading github.com/lucasb-eyer/go-colorful v1.0.2
go: downloading github.com/microsoft/azure-devops-go-api/azuredevops v0.0.0-20191014190507-26902c1d4325
go: downloading github.com/docker/distribution v2.7.1+incompatible
go: downloading github.com/digitalocean/godo v1.42.1
go: downloading github.com/google/go-querystring v1.0.0
go: downloading google.golang.org/api v0.30.0
go: downloading github.com/wtfutil/spotigopher v0.0.0-20191127141047-7d8168fe103a
go: downloading code.cloudfoundry.org/bytefmt v0.0.0-20190819182555-854d396b647c
go: downloading sigs.k8s.io/yaml v1.1.0
go: downloading github.com/shirou/gopsutil v2.20.7+incompatible
go: downloading github.com/PagerDuty/go-pagerduty v0.0.0-20191002190746-f60f4fc45222
go: downloading gopkg.in/yaml.v2 v2.3.0
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/json-iterator/go v1.1.7
go: downloading github.com/godbus/dbus v4.1.0+incompatible
go: downloading github.com/cenkalti/backoff v2.2.1+incompatible
go: downloading github.com/dlclark/regexp2 v1.2.0
go: downloading github.com/hekmon/transmissionrpc v0.0.0-20190525133028-1d589625bacd
go: downloading github.com/wtfutil/todoist v0.0.2-0.20191216004217-0ec29ceda61a
go: downloading github.com/stretchr/testify v1.6.1
go: downloading google.golang.org/appengine v1.6.6
go: downloading github.com/opencontainers/go-digest v1.0.0-rc1
go: downloading github.com/google/uuid v1.1.1
go: downloading cloud.google.com/go v0.62.0
go: downloading github.com/adlio/trello v1.7.0
go: downloading github.com/zmb3/spotify v0.0.0-20191010212056-e12fb981aacb
go: downloading github.com/danwakefield/fnmatch v0.0.0-20160403171240-cbb64ac3d964
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading github.com/googleapis/gnostic v0.1.0
go: downloading gotest.tools v2.2.0+incompatible
go: downloading github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e
go: downloading github.com/mmcdole/gofeed v1.0.0
go: downloading github.com/gregjones/httpcache v0.0.0-20190212212710-3befbb6ad0cc
go: downloading github.com/golang/protobuf v1.4.2
go: downloading github.com/hekmon/cunits v2.0.1+incompatible
go: downloading github.com/docker/go-units v0.4.0
go: downloading gopkg.in/inf.v0 v0.9.1
go: downloading google.golang.org/protobuf v1.25.0
go: downloading github.com/mmcdole/goxpp v0.0.0-20181012175147-0068e33feabf
go: downloading github.com/jessevdk/go-flags v1.4.0
go: downloading github.com/google/go-cmp v0.5.1
go: downloading gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
go: downloading github.com/Microsoft/go-winio v0.4.14
go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
go: downloading github.com/peterbourgon/diskv v2.0.1+incompatible
go: downloading github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/gophercloud/gophercloud v0.5.0
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/nicklaw5/helix v0.5.9
go: downloading github.com/PuerkitoBio/goquery v1.5.0
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading github.com/andybalholm/cascadia v1.0.0
go: downloading github.com/google/btree v1.0.0
go: downloading google.golang.org/grpc v1.29.1
go: downloading github.com/go-ole/go-ole v1.2.4
go: downloading github.com/googleapis/gax-go/v2 v2.0.5
go: downloading go.opencensus.io v0.22.4
go: downloading github.com/Azure/go-autorest v11.1.2+incompatible
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
go: downloading google.golang.org/genproto v0.0.0-20200804131852-c06518451d9c
go: downloading github.com/ovh/venom v0.25.0
go: downloading github.com/sguiheux/go-coverage v0.0.0-20190710153556-287b082a7197
go: downloading github.com/sirupsen/logrus v1.4.2
go: downloading github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: downloading gopkg.in/AlecAivazis/survey.v1 v1.7.1
go: downloading github.com/fsamin/go-dump v1.0.9
go: downloading github.com/fatih/color v1.7.0
go: downloading github.com/spf13/cobra v0.0.5
go: downloading github.com/pborman/uuid v1.2.0
go: downloading github.com/mattn/go-isatty v0.0.12
go: downloading github.com/maruel/panicparse v1.3.0
go: downloading github.com/go-gorp/gorp v2.0.0+incompatible
go: downloading github.com/mitchellh/hashstructure v0.0.0-20170609045927-2bca23e0e452
go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.2
go: downloading github.com/eapache/go-resiliency v1.1.0
go: downloading github.com/spf13/afero v1.2.2
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: downloading github.com/ovh/cds/sdk/interpolate v0.0.0-20190319104452-71125b036b25
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/mattn/go-colorable v0.1.6
go: downloading github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
go: downloading github.com/mndrix/tap-go v0.0.0-20170113192335-56cca451570b
go: downloading github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading github.com/aokoli/goutils v1.1.0
go: downloading github.com/huandu/xstrings v1.2.0
installing
hash mismatch in fixed-output derivation '/nix/store/cbn96y1r2h21a9zrsrr1ffgm25nlbcgg-wtf-0.32.0-go-modules':
  wanted: sha256:0l1q29mdb13ir7n1x65jfnrmy1lamlsa6hm2jagf6yjbm6wf1kw4
  got:    sha256:0p4cy7snxkx5in7c0nsy6bm0f1gn297rzwalm5rdfvhgdlb102nw
cannot build derivation '/nix/store/hfkjz60aksbgy7x9xmrbbbi30rjmzmp7-wtf-0.32.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/hfkjz60aksbgy7x9xmrbbbi30rjmzmp7-wtf-0.32.0.drv' failed
```

**Comment by aanderse** [comment]
@GrahamcOfBorg build wtf

**Comment by avdv** [comment]
> wtf on x86_64-darwin Queued — Waiting to run this check... Started 7d 13h 20m 46s ago 

I guess it got stuck and we can call this a failure?! What next?

**Comment by stigtsp** [comment]
f7e76d181e22b0 is failing to build for me. I propose we mark it as `broken = stdenv.isDarwin`

```
% nix-build -A wtf
these derivations will be built:
  /nix/store/f29hm2393h56xh4slil62zlb72j5k0nq-wtf-0.32.0.drv
building '/nix/store/f29hm2393h56xh4slil62zlb72j5k0nq-wtf-0.32.0.drv'...
unpacking sources
unpacking source archive /nix/store/7xd1c92n7i0pcpxb71qgfz7d0hasbcrf-source
source root is source
patching sources
configuring
building
Building subPackage ./.
modules/azuredevops/widget.go:7:2: cannot find package "." in:
        /private/var/folders/hk/77b9kbln51v8hgc1c6j767k00000gn/T/nix-build-wtf-0.32.0.drv-0/source/vendor/github.com/microsoft/azure-devops-go-api/azuredevops
modules/azuredevops/client.go:7:2: cannot find package "." in:
        /private/var/folders/hk/77b9kbln51v8hgc1c6j767k00000gn/T/nix-build-wtf-0.32.0.drv-0/source/vendor/github.com/microsoft/azure-devops-go-api/azuredevops/build
builder for '/nix/store/f29hm2393h56xh4slil62zlb72j5k0nq-wtf-0.32.0.drv' failed with exit code 1
error: build of '/nix/store/f29hm2393h56xh4slil62zlb72j5k0nq-wtf-0.32.0.drv' failed
```

**Comment by avdv** [comment]
> I propose we mark it as `broken = stdenv.isDarwin`

Done.

I also removed the conditional vendor sha256 computation again and force pushed to the branch, original branch still available [here](https://github.com/avdv/nixpkgs/tree/auto-update/wtf_v1).

File: pkgs/applications/misc/wtf/default.nix [hunk 1]
```diff
@@ -37,5 +38,6 @@ buildGoModule rec {
     license = licenses.mpl20;
     maintainers = with maintainers; [ kalbasit ];
     platforms = platforms.linux ++ platforms.darwin;
+    broken = stdenv.isDarwin
```
Comment (stigtsp) [inline pkgs/applications/misc/wtf/default.nix:41]:
```suggestion
    broken = stdenv.isDarwin;
```
Missing trailing semicolon

**Review by stigtsp — CHANGES_REQUESTED** [review]

**Comment by stigtsp** [comment]
@avdv The commit message on b3fc088 doesn't match [CONTRIBUTING.md#submitting-changes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes), either update the commit message to something like `wtf: broken on darwin`

**Comment by stigtsp** [comment]
@GrahamcOfBorg build wtf

**Review by stigtsp — APPROVED** [review]
LGTM, thanks for fixing all the nitpicks :-)

Result of `nixpkgs-review pr 98622` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>wtf</li>
  </ul>
</details>

**Pull request merged**